### PR TITLE
[FSTORE-519] update redshift documentation

### PR DESCRIPTION
**Pull request by DhananjayMukhedkar**

### CODE REVIEW

**Comment by DhananjayMukhedkar** [comment]
JIRA https://hopsworks.atlassian.net/browse/FSTORE-519

File: docs/user_guides/fs/storage_connector/creation/redshift.md [hunk 1]
```diff
@@ -20,7 +20,10 @@ Before you begin this guide you'll need to retrieve the following information fr
...
+- **Authentication method:** There are two options available for authenticating with the Redshift cluster. The first option is to configure a username and a password. 
+The second option is to configure an IAM role. With IAM roles, Jobs or notebooks launched on Hopsworks do not need to explicitly authenticate with Redshift, as the HSFS library will transparently use the IAM role to acquire a temporary credential to authenticate the specified user. 
+Read more about IAM roles in our [AWS credentials passthrough guide](../../../../admin/roleChaining.md). Note, the option `Instance Role` allows the user to save the connector.
+  However, this option does not yet support to read from the connector.
```
Comment (jimdowling) [inline docs/user_guides/fs/storage_connector/creation/redshift.md:26]: However, this option does not yet support reading from the connector.

File: docs/user_guides/fs/storage_connector/creation/redshift.md [hunk 2]
```diff
@@ -38,15 +41,17 @@ Enter the details for your Redshift connector. Start by giving it a **name** and
 
 1. Select "Redshift" as connector protocol.
 2. The name of the cluster.
-3. The name of the database driver class.
-4. The database endpoint.
-5. The database name.
-6. The database port.
-7. The database username, here you have the possibility to let Hopsworks auto-create the username for you.
+3. The database endpoint. Should be in the format of `[UUID].eu-west-1.redshift.amazonaws.com`. For example, if the endpoint info
```
Comment (jimdowling) [inline docs/user_guides/fs/storage_connector/creation/redshift.md:44]: Should be in the format `[UUID].eu-west-1.redshift.amazonaws.com`

File: docs/user_guides/fs/storage_connector/creation/redshift.md [hunk 3]
```diff
@@ -38,15 +41,17 @@ Enter the details for your Redshift connector. Start by giving it a **name** and
...
+   displayed in Redshift is `cluster-id.uuid.eu-north-1.redshift.amazonaws.com:5439/dev` the value to enter 
+   here is just `uuid.eu-north-1.redshift.amazonaws.com` 
+4. The database name.
+5. The database port.
+6. The database username, here you have the possibility to let Hopsworks auto-create the username for you.
 8. Database Driver (optional): You can use the default JDBC Redshift Driver `com.amazon.redshift.jdbc42.Driver` 
    included in Hopsworks or set a different driver (More on this later).
-9. Optionally provide the database group and table to point the connector to.
-9. Set the appropriate authentication method.
+9. Optionally provide the database group and table to point the connector to. A database group is the group created
```
Comment (jimdowling) [inline docs/user_guides/fs/storage_connector/creation/redshift.md:52]: Don't finish a sentence on a preposition 'to'

 Optionally provide the database group and table as the target of the connector.
 or
 Optionally provide the database group and table for the connector.

File: docs/user_guides/fs/storage_connector/creation/redshift.md [hunk 4]
```diff
@@ -59,38 +64,40 @@ Enter the details for your Redshift connector. Start by giving it a **name** and
...
-
 #### Add the driver to Jupyter Notebooks and Spark jobs
 
 You can now add the driver file to the default job and Jupyter configuration. This way, all jobs and Jupyter instances in the project will have the driver attached so that Spark can access it.
 
 1. Go into the Project's settings.
 2. Select "Compute configuration".
 3. Select "Spark".
-4. Select and Upload the jar file in the "additional jars" section of the configuration.
+4. Under "Additional Jars" choose to "Upload new file" to upload the driver jar file.
```
Comment (jimdowling) [inline docs/user_guides/fs/storage_connector/creation/redshift.md:79]: choose "Upload new file"

File: docs/user_guides/fs/storage_connector/creation/redshift.md [hunk 5]
```diff
@@ -59,38 +64,40 @@ Enter the details for your Redshift connector. Start by giving it a **name** and
...
 
 <figure markdown>
-  ![Redshift Driver Job and Jupyter Configuration](../../../../assets/images/guides/fs/storage_connector/driver_upload.png)
+  ![Redshift Driver Job and Jupyter Configuration](../../../../assets/images/guides/fs/storage_connector/jupyter_config.png)
   <figcaption>Attaching the Redshift Driver to all Jobs and Jupyter Instances of the Project</figcaption>
 </figure>
 
+Alternatively, you can choose "From Project" option to choose an already existing jar file in the Project. For this,
```
Comment (jimdowling) [inline docs/user_guides/fs/storage_connector/creation/redshift.md:86]: Alternatively, you can choose the "From Project" option

File: docs/user_guides/fs/storage_connector/creation/redshift.md [hunk 6]
```diff
@@ -59,38 +64,40 @@ Enter the details for your Redshift connector. Start by giving it a **name** and
...
+4. Under "Additional Jars" choose to "Upload new file" to upload the driver jar file.
 
 <figure markdown>
-  ![Redshift Driver Job and Jupyter Configuration](../../../../assets/images/guides/fs/storage_connector/driver_upload.png)
+  ![Redshift Driver Job and Jupyter Configuration](../../../../assets/images/guides/fs/storage_connector/jupyter_config.png)
   <figcaption>Attaching the Redshift Driver to all Jobs and Jupyter Instances of the Project</figcaption>
 </figure>
 
+Alternatively, you can choose "From Project" option to choose an already existing jar file in the Project. For this, 
+you will first have to upload a jar file in Project using the File Browser if not already, and then choose the
```
Comment (jimdowling) [inline docs/user_guides/fs/storage_connector/creation/redshift.md:87]: you will first have to upload the jar file to the Project using the File Browser. After you have uploaded the jar file, you can select if using the "From Project" option.

File: docs/user_guides/fs/storage_connector/creation/redshift.md [hunk 7]
```diff
@@ -59,38 +64,40 @@ Enter the details for your Redshift connector. Start by giving it a **name** and
...
 
 <figure markdown>
-  ![Redshift Driver Job and Jupyter Configuration](../../../../assets/images/guides/fs/storage_connector/driver_upload.png)
+  ![Redshift Driver Job and Jupyter Configuration](../../../../assets/images/guides/fs/storage_connector/jupyter_config.png)
   <figcaption>Attaching the Redshift Driver to all Jobs and Jupyter Instances of the Project</figcaption>
 </figure>
 
+Alternatively, you can choose "From Project" option to choose an already existing jar file in the Project. For this, 
+you will first have to upload a jar file in Project using the File Browser if not already, and then choose the 
+desired file using the "From Project" option. To upload the jar file in Project through File Browser, see below example:
```
Comment (jimdowling) [inline docs/user_guides/fs/storage_connector/creation/redshift.md:88]: To upload the jar file to the Project through the File Browser, see the example below:

File: docs/user_guides/fs/storage_connector/creation/redshift.md [hunk 8]
```diff
@@ -59,38 +64,40 @@ Enter the details for your Redshift connector. Start by giving it a **name** and
...
+
+   1. Open File Browser
+   2. Navigate to "Resources" directory
+   3. Upload the jar file
+
+<figure markdown>
+  ![Redshift Driver Upload](../../../../assets/images/guides/fs/storage_connector/driver_upload.png)
+  <figcaption>Redshift Driver Upload in the Filesystem Browser</figcaption>
```
Comment (jimdowling) [inline docs/user_guides/fs/storage_connector/creation/redshift.md:96]: You call it "File Browser", not "Filesystem Browser" in other places.

**Review by jimdowling — CHANGES_REQUESTED** [review]
Typos

**Comment by DhananjayMukhedkar** [comment]
fixed above typos

**Review by jimdowling — APPROVED** [review]

**Pull request merged**